### PR TITLE
Removes plastic reaction, adds coal/plastic ore.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
Finally, my FINAL PLASTIC PACK. It's my STRONGEST ATTACK.

## About The Pull Request

First off, this removes the plastic chemistry reaction, and the plastic polymers chemical as a result (Which did nothing on it's own anyway). Overall, it's healthier for the game as plastic was an infinitely attainable material that only required power and chemistry access, which makes it difficult to balance around materials.

Second, this adds coal ore to all mining sources, as a low value but decently common ore in the regular random pool, with better odds of showing up in low value rocks. It can be refined to generate plastic sheets from the ORM, or, used in hand to generate sheets of coke fuel, which WAS the coal used in grills/charcoal/etc, and has been renamed to avoid confusion. It may still be a bit confusing, which is why I gave coke a new unique icon to help keep the two distinct.

## Why It's Good For The Game

Infinite materials are unhealthy for balance reasons, and because plastic now has a use in every department now in some way, this allows at least some small amount of plastic to be generated each round if you even have access to miners and minerals in the first place. Plus, charcoal and plastic based objects will be attainable slightly more often within rounds.

## Changelog
:cl:
add: Adds Coal ore to Lava/Iceland mining.
add: Coal ore can be refined into plastic at the ORM, or converted into coke by hand.
del: The Plastic chemistry reaction/Plastic Polymers has been removed.
tweak: The OLD coal (used by grills) has been renamed to coke, and given a new sprite.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
but why

**Comment by ATH1909** [comment]
Why would you turn a material with a unique acquisition method that can serve as something to keep plumbers occupied for a few minutes into just another ore?

**Comment by ATH1909** [comment]
image

**Comment by wesoda25** [comment]
The plastic reaction sucks especially considering how many uses plastic is getting, this is overdue. That being you're wasting the opportunity to add oil (not makeable) deposits to lavaland that people can harvest and turn into plastic

**Comment by HomocidlePancakes** [comment]
The plastic reaction sucks because noone can be arsed to do it.

**Comment by ATH1909** [comment]
> The plastic reaction sucks especially considering how many uses plastic is getting, this is overdue. That being said you're wasting the opportunity to add oil (not makeable) deposits to lavaland that people can harvest and turn into plastic

those are already in the game, by the way- the elephant graveyard ruin uses them

**Comment by Tlaltecuhtli** [comment]
>infinite materials are unhealthy
>makes plastic titanium 2.0

no one is gonna notice the lack of infinity once 10 min have passed and the lathe has 5 stacks of it

**Comment by Picklesoup** [comment]
If this is an attempt to balance plastic because it has an export price, wouldn't it make more sense to remove the export price instead of changing the entire material and the method you obtain it? There's not even many things to do with it besides that.

Also, it is trivially easy to pull out near infinite amounts of metal from the ORM or lathes because metal is so common. If people want to construct stuff, I'm certain they're using metal and not plastic.

**Comment by Tlaltecuhtli** [comment]
> Wouldn't it make more sense to remove the export price instead of changing the entire material and the method you obtain it? You can just keep in mind how easy it is to obtain while making stuff that uses it, like metal.

the "value" is like less than a credit for sheet, export value was never a problem its just some weird thing people think that people actually export plastic for mad millions

**Comment by Picklesoup** [comment]
Also, can you elaborate on what you mean by "unhealthy for balance reasons"? I'm not exactly sure what you mean by that, it's very vague.

**Comment by ArcaneMusic** [comment]
> Also, can you elaborate on what you mean by "unhealthy for balance reasons"? I'm not exactly sure what you mean by that, it's very vague.



> Why would you turn a material with a unique acquisition method that can serve as something to keep plumbers occupied for a few minutes into just another ore?

Alright so I'll elaborate more on this because I've talked about this at length in the past and in other PRs and forum threads. in particular, I talked about this on [THIS](https://tgstation13.org/phpBB/viewtopic.php?f=10&t=26416) thead, but I'll go over the key points.

Plastic as it stands right now is in a horrible place for both the material economy and cash economy, as it's been given an export value, and it's required for a select few unique things that need it to be made in a lathe/crafting. Thing is, if you can create plastic between 2 roundstart chemistry machines in a matter of seconds, that's both tedious and also doesn't generate scarcity unless the entire station's power is killed. For reference, scarcity is the backbone of all the ingame economies, regardless of how broken cargo may be with hydrogen exports, or station income, etc. We wouldn't lock cool shit behind diamond if diamond could be obtained with no effort. Plus, as someone who has set up plastic plumbing factories dozens of times, it enables you to make infinite amounts of objects that can fill dozens of station niches (extra storage, furniture, stealth objects, you name it).

Now, if you wanted to do something along the lines of refining oil into plastic as it's own self-serving resource, to meet those same balance requirements for scarcity, you'd have to limit oil in some capacity. And by doing that... you'd stand to make chemistry kinda unplayable considering that all advanced healing/production chemicals use oil in some fashion. Alternatively you'd have to make a new reagent that meets scarcity standards, but currently the only 2 departments that can do that effectively is mining, or botany, and at least with mining, it's far more consistent when you see results.

tl;dr: Having infinite of anything devalues everything it touches, and means either mass-nerfing down the line, or it would mean a treatment similar to this.

**Comment by Capsandi** [comment]
How about replacing the ash or oil in the reaction with cellulose fibers? You can't mass produce those.
From wikipedia:
>They(Plastics) are usually synthetic, most commonly derived from petrochemicals, however, **an array of variants are made from renewable materials such as polylactic acid from corn or cellulosics**

**Comment by necromanceranne** [comment]
Who cares about it being mass producable, I care more about it showing up at all. Rarely do chemists bother, and this seems like a fine way to intergrate it into the rest of our materials.

**Comment by XDTM** [comment]
i'm all for making it a resource that needs to be acquired instead of generated from nothing, but it'd be nice if it was from oil wells and some kind of oil pump machinery, making material gathering from lavaland a bit more varied.

**Comment by ATH1909** [comment]
> Plus, as someone who has set up plastic plumbing factories dozens of times, it enables you to make infinite amounts of objects that can fill dozens of station niches (extra storage, 

If you're talking about plastic boxes, you do know that you can just empty out any box (or medkit) of, say, bodybags or glasses (but not lightbulbs; their boxes are special and need to be flattened and remade into normal boxes in order to behave like most other boxes do) and use that as a box, right? Being able to make boxes isn't very special (especially when cargo can order 50 sheets of cardboard for, like, 2,000 credits (I might be way off there; I haven't checked the price in a while)).

> furniture, 

Oh no, the powergamers are making cheap plastic furniture for aesthetic reasons, whatever shall we do?

> stealth objects

You mean those plastic plants that are in practically every room on the station?

Plastic is an easily mass-produceable material, so you can use it to make mass-produceable trash/low-tier items. I fail to see the problem here.

**Comment by ExcessiveUseOfCobblestone** [comment]
If the argument is theoretically infinite materials, is it really that big of an issue compared to lavaland? I have never seen a round where miners have a hard time finding specific materials if they cared to mine, but I've seen plenty where we have enough materials to buy everything for whoever wants it and sell enough to get whatever you wanted (barring meme bike), which is pretty much why you wouldn't want infinite to begin with. I don't disagree theoretically infinite is bad, I am just not thrilled with the fact that moving it to mining doesn't really change the end result that brings people to dislike infinite resources to begin with.

If the argument is players not doing it, I don't really want to fix that by moving the interaction out of the job (especially since I don't think I've ever even been asked for plastic as a chemist). Combatting meth makers by taking out "valuable" items that they could have made besides meth so their encouraged to just continue making meth seems kinda odd.

**Comment by EOBGames** [comment]
The infinite material argument is an odd one, the real reason a PR like this would be helpful is so we can actually see some damned plastic sometime. I can count the number of times I've actually seen chem make it and put it in the lathe on one hand, let alone times it's been made in a large enough quantity to make anything more than 2 bottles and a colo cup.

**Comment by Tlaltecuhtli** [comment]
heh the intermediate beakers that need plastic are usually "not needed", everyone just uses muscle memory chem recipes for 100u or less.
 you make plastic for a large bottle if you happen in a situation where you quickly need a chem and after breaking into chemistry you dont find any 100u beaker so you "upgrade" to 100u by making plastic in an 50u or 30u bottle, which is a neat trick
only times i ever needed plastic was to make a plumbing constructor as assistant and i would just decon plastic flaps next to the lathe to print one
plastic has literally 0 uses in mass quntity, if someone in future makes  plastic more needed then the worst thing you could do is to turn it into titanium 2.0, the epic mineral for super high tech things that is literally more common than plasma ore , its like having a yu gi oh deck need 20 different cards but you get all the cards you need from the same pack on amazon for 4.99$

**Comment by necromanceranne** [comment]
I just deconstruct plastic flaps and infuriate service since now all their flaps are gone to make my extra large beakers full of healmix and surgical drapes

**Comment by Learner10** [comment]
In fairness, if you need plastic, you could go to the chemist, and ask for plastic. It's not like you just asked them to make C3's. It's simple to make.

**Comment by ExcessiveUseOfCobblestone** [comment]
It's actually a bit harder since if you are making ash via oil, you have to make sure you add the acid in between to cool it down so the oil won't also turn into ash.

Then again talking about chem complexity for things that can be made entirely in the dispenser is a bit...

**Comment by ArcaneMusic** [comment]
Welp, I feel like this has been a universally hot take, somehow, so I guess I'll close this for now when the world needs it most.

**Comment by ArcaneMusic** [comment]
@ExcessiveUseOfCobblestone What are your thoughts about this in modern day year of our lord 2022?

**Comment by Iamgoofball** [comment]
i would like to note the only place I see plastic produced by chemistry regularly is the downstream

**Comment by RikuTheKiller** [comment]
I personally do make plastic as a chemist, but only if I need bigger beakers or when someone asks me to. Genuinely, though, due to how low quality the items are I don't think we should really be making plastic more bland. It's the ONLY concrete resource chemistry can produce. (I've made plastic factories capable of making millions of units of plastic. Even then I've never seen it abused by the QM for some reason, so I don't think it's a big issue. Mostly was just used for beakers, medical items and maybe some random heretic got hold of a plastic knife. Same thing would've happened if I just made 10 sheets.)

**Comment by Learner10** [comment]
During more desperate times while an engineer strapped for iron, I've gone down the dark road of replacing walls with wood, pizza, meat, and yes, even plastic.
